lib/rspecq/worker.rb

```diff
@@ -15,16 +15,17 @@ class Worker
     # will be split and scheduled on a per-example basis.
     attr_accessor :file_split_threshold
 
-    attr_reader :queue
+    attr_reader :queue, :max_requeues
```

Comment:
It'd be nice to have this documented - you could reuse the documentation of `MAX_REQUEUES`.